galera_k8s_v1.6.yml

```diff
@@ -28,10 +28,15 @@ spec:
       annotations:
         pod.alpha.kubernetes.io/initialized: "true"
     spec:
+      securityContext:
+        runAsUser: 27
+        fsGroup: 0
```

Comment:
It would but after testing it does seem that the container is actually run with group 0:
```
> docker build -t galtest . && \
   docker run -d --name galtestrun galtest && \
   docker exec -it galtestrun ps xo pid,cmd,uid,group
```
...build output...
```
  PID CMD                           UID GROUP
    1 /bin/bash /usr/bin/containe    27 root
    7 /bin/bash /usr/share/contai    27 root
    8 /bin/sh /usr/bin/mysql_inst    27 root
   28 /bin/sh /usr/bin/mysql_inst    27 root
   31 /usr/sbin/mysqld --lc-messa    27 root
   32 ps xo pid,cmd,uid,group        27 root
```

Reply:
Happy to add GID to the dDockerfile if you prefer? https://github.com/adfinis-sygroup/openshift-mariadb-galera/blob/master/k8s-mariadb-galera-centos/Dockerfile#L30

Reply:
Wasn't aware that one had to specify the GID too when using `USER`. Feel free to test the change to `USER 27:27` and change to `fsGroup: 27` if it works afterwards.